### ISSUE TITLE
Bug with latest version?

### ISSUE THREAD

**Issue by darbymanning** [ganlanyuan/tiny-slider#61]
Since updating to the latest release I'm not able to import the module into my webpack project. 

**Example code:**
`import { tns } from "tiny-slider/src/tiny-slider.module"`

**Output:**

```
application.js:15239 Uncaught ReferenceError: prop is not defined
    at whichProperty (application.js:15239)
    at Object.keys.keys (application.js:12477)
    at __webpack_require__ (application.js:677)
    at fn (application.js:88)
    at Object.<anonymous> (application.js:12176)
    at __webpack_require__ (application.js:677)
    at fn (application.js:88)
    at Object.window.KBA (application.js:10295)
    at __webpack_require__ (application.js:677)
    at fn (application.js:88)
```

**Trace:**
```
/***/ }),
/* 137 */
/***/ (function(module, __webpack_exports__, __webpack_require__) {

"use strict";
/* harmony export (immutable) */ __webpack_exports__["a"] = whichProperty;
function whichProperty(props) {
  var el = document.createElement('fakeelement'),
      len = props.length;
  for (var i = 0; i < props.length; i++) {
    prop = props[i];
    if (el.style[prop] !== undefined) {
      return prop;
    }
  }

  return false; // explicit for ie9-
}
```

**Comment by Muskos** [comment]
+1

**Comment by ganlanyuan** [comment]
This should be fixed in v2.1.2
Would you be able to check it?

**Comment by mkoehler2911** [comment]
Works! Great and fast job thank you :)
